.infrastructure/todoapp-pod.yml

```diff
@@ -3,12 +3,21 @@ kind: Pod
 metadata:
   name: todoapp
   namespace: todoapp
+  labels:
+    app: todoapp
 spec:
   containers:
   - name: todoapp
     image: ikulyk404/todoapp:3.0.0
     ports:
     - containerPort: 8080
+    env:
+    - name: PYTHONUNBUFFERED
+      value: "1"
+    - name: DJANGO_SETTINGS_MODULE
+      value: myproject.settings
+    - name: DATABASE_URL
+      value: postgres://user:password@host:5432/mydb
```

Comment:
What's the point of these variables if they're not in the code, and the program doesn't read them?